src/chainstate/stacks/index/marf.rs

```diff
@@ -68,6 +68,57 @@ struct WriteChainTip<T> {
     height: u32,
 }
 
+/// Options for opening a MARF
+#[derive(Clone, Debug)]
+pub struct MARFOpenOpts {
+    /// Hash calculation mode for calculating a trie root hash
+    pub hash_calculation_mode: TrieHashCalculationMode,
+    /// Cache strategy to use
+    pub cache_strategy: String,
+    /// store trie blobs externally from the DB, in a flat file
+    pub external_blobs: bool,
+    /// unconditionally do a DB migration (used for testing)
+    pub force_db_migrate: bool,
+}
+
+impl MARFOpenOpts {
+    pub fn default() -> MARFOpenOpts {
+        MARFOpenOpts {
+            hash_calculation_mode: TrieHashCalculationMode::Deferred,
+            cache_strategy: "noop".to_string(),
```

Comment:
Does the `default` method in the cache strategy use a different default? That's reading an environment variable.

Reply:
I'm not sure I understand the question?  The reason `default()` uses `"noop"` here, instead of reading the environment variable, is because the environment variable is the concern of the cache options, not the MARF options.

Reply:
But isn't the cache option for a MARF ultimately being controlled by the `MARFOpenOpts`?

Reply:
Like, if I wanted to enable caching for the Clarity MARF, how would that work? The `MARFOpenOpts` are specified by invoking `MARFOpenOpts::default()` and then setting the storage strategy to `TrieFile` storage, but the cache strategy will always be `noop`, regardless of the environment variable setting.

Reply:
I instead made it so that the `MARFOpenOpts` passed to the Stacks chainstate is specified in the `stacks-node::config::Config` struct.